maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -24070,9 +24070,7 @@
 	},
 /area/almayer/living/cryo_cells)
 "bAZ" = (
-/obj/structure/machinery/cm_vending/clothing/vehicle_crew{
-	pixel_x = 32
-	},
+/turf/open/space/basic,
```

Comment:
space? What you are meant to do is move the vendor in by pixel shifting it East and changing the density similar to the other vendors. There is still meant to be a wall below it.

Reply:
Yeah I think I messed something up, will fix